src/java/com/zimbra/graphql/resolvers/impl/AccountResolver.java

```diff
@@ -96,4 +98,16 @@ public void accountEndSession(
         @GraphQLRootContext RequestContext context) throws ServiceException {
         return accountRepository.prefsModify(context, prefs);
     }
+
+    @GraphQLMutation(description="Change password.")
+    public ChangePasswordResponse changePassword(
+        @GraphQLArgument(name = GqlConstants.ACCOUNT_SELECTOR, description =
```

Comment:
For consistency, keep these params as single line each.

Reply:
I  understand the need for consistency but having line greater than 80-90 characters makes it difficult to read.